### ISSUE TITLE
Linear learner Logistic Regression predict probability

### ISSUE THREAD

**Issue by peterkim95** [aws/sagemaker-python-sdk#748]
I am using Linear Learner with {predictor_type = 'binary classifier', loss='logistic'}. During prediction, is there a way to access the predicted probability instead of the final label? 

E.g. Sklearn can do this as such: https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html#sklearn.linear_model.LogisticRegression.predict_proba

Is there something similar for Sagemaker?

**Comment by cemsaz** [comment]
@peterkim95, I am looking at the same problem, is there any solution to this?

**Comment by brandon-segal** [comment]
@peterkim95  I am looking for a similar solution for a sklearn container that I am testing. The deployed model only has a predict method but no predict_proba method similar to the sklearn model
